### PR TITLE
Althome

### PR DESCRIPTION
**Pull request by danielfang97**
added althome to theme, working on sbh2 local where you can click the top left logo and redirect to whatever you set as the alternate homepage

### CODE REVIEW

File: frontend/specialAccess/setup.js [hunk 1]
```diff
@@ -240,18 +246,18 @@ export default function Setup({ setInSetupMode }) {
...
                 }
               );
               setErrors([]);
               setInSetupMode(false);
             } catch (error) {
-              if (error.response.status === 400) {
+              if (error.response?.status === 400) {
                 const errorMessages = error.response.data.details.map(
-                  error => error.message
+                  (err) => err.message
```
Comment (cjmyers) [inline frontend/specialAccess/setup.js:260]: Is this correct?  Seems like it should be error?

**Review by cjmyers — COMMENTED** [review]
See comment

**Comment by danielfang97** [comment]
I think chatgpt changed some syntax, and I didn't notice. I'll change it back. Thanks for catching it. You probably need to approve it first, and then I can make a new branch to change it since there's some other branches that's making it so I can't push it to Github.

**Comment by cjmyers** [comment]
Please be more careful with ChatGPT.  I will approve this one now.

**Review by cjmyers — APPROVED** [review]

**Comment by cjmyers** [comment]
Merged by habit.  Please get the fix in through another PR quickly.

**Pull request merged**